ui/client/components/Tips.js

```diff
@@ -53,26 +43,116 @@ export class Tips extends Component {
     }
   }
 
-  printError = (error, suffix, idx) => {
-    return (
-      <div key={idx + suffix} title={error.description}>
-      {(suffix ? suffix + ": " : '') + error.message + (error.fieldName ? `(${error.fieldName})` : "")}
+  errors = (errors) =>
+    <div key={uuid4()} className={"error-tips"}>
+      {this.headerIcon(errors)}
```

Comment:
What do you think about creating two components like: GlobalErrors and GlobalWarning, where we can move all logic with parsing and presenting data? Now we have all these in one component - I think it's too large :)

Reply:
Done